src/sequence/mod.rs

```diff
@@ -745,7 +745,7 @@ mod tests {
     }
 
     // Generate a vec of Sequence replicas of some length
-    fn generate_replicas(max_quantity: usize) -> impl Strategy<Value = Vec<Sequence>> {
+    fn gen_replicas(max_quantity: usize) -> impl Strategy<Value = Vec<Sequence>> {
```

Comment:
IMO `generate` > `gen` for clarity. (general replicas?) I know it's a stretch... but IMO it's one less thing to trip folk up.